statsmodels/graphics/regressionplots.py

```diff
@@ -822,8 +822,6 @@ def plot_leverage_resid2(results, alpha=.05, label_kwargs={}, ax=None,
     alpha : float
         Specifies the cut-off for large-standardized residuals. Residuals
         are assumed to be distributed N(0, 1) with alpha=alpha.
-    label_kwargs : dict
-        The keywords to pass to annotate for the labels.
```

Comment:
my first response in cases like this is to ask whether there is a missing extension that was planned but never finished. Based on the PR #558 it looks like only the partial regression plot got working label_kwargs.
I don't really know what the annotate options are doing.